ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Entity.java

```diff
@@ -0,0 +1,31 @@
+package com.github.migi_1.Context.model.entity;
+
+/**
+ * Superclass of all object that implement some MovableBehaviour.
+ *
+ * @author Marcel
+ *
+ */
+public abstract class Entity {
```

Comment:
Entity should be the one that implements IMovable, so that the other ones don't have to.